app/src/main/java/com/brainwallet/data/model/Language.kt

```diff
@@ -25,6 +27,16 @@ enum class Language(
     ;
 
     companion object {
-        fun find(code: String?): Language = values().find { it.code == code } ?: ENGLISH
+        fun find(code: String?): Language = entries.find { it.code == code } ?: ENGLISH
+    }
+
+    fun toLocale(): Locale {
```

Comment:
Looks nice and neat @andhikayuana . Does this work for all cases? Does it work for xhants and Ru?  We had issues in the past